markdown_anki_decks/cli.py

```diff
@@ -242,10 +248,9 @@ def read_file(file):
 
 
 # check if a file is a markdown file
-def is_markdown_file(file):
-    """Check if a file is a markdown file."""
-    # TODO(lukemurray): parameterize markdown extensions?
-    return file.endswith(".md")
+def is_correct_filetype(file, filetype: str):
```

Comment:
change `is_correct_filetype` to `has_file_extension`.

 change the `filetype` parameter to `extension` or `extensions` (up to you). Also the second parameter will be a list once you make the changes from above so you'll have to change some of the types and logic here.